include/arch/arm-imx/limits.h

```diff
@@ -49,10 +49,11 @@
 
 #define SSIZE_MAX INT_MAX
 
-#define PTHREAD_STACK_MIN 256
-#define PATH_MAX 1024
-#define ARG_MAX 128
+#define PTHREAD_STACK_MIN  256
+#define PATH_MAX           1024
+#define ARG_MAX            128
 #define _POSIX2_RE_DUP_MAX 255
+#define HOST_NAME_MAX      255
```

Comment:
Kernel uses this value - it should be defined in kernel. Libphoenix uses kernel header, not vice versa

Reply:
OK, where do do think it should go in the kernel? I have to include kernel header because HOST_NAME_MAX must be available in "limit.h" (POSIX).

Reply:
This file looks perfect for this: https://github.com/phoenix-rtos/phoenix-rtos-kernel/blob/master/include/posix.h

Reply:
All constants and structures shared with kernel should be defined in this directory

Reply:
@agkaminski remember that C and POSIX requirees that you avoid unneeded defines in headers (underscore and big letter or another underscore is allowed). You cannot easy include kernel posix.h in limits.h because you pollute limits.h header and viiolate rule about reserved identifiers.

Reply:
Ok, then maybe we need to create limits.h in kernel include? This would solve this problem.

Reply:
I agree with that. It seems as a better approach